backend/internal/agents/librarian.go

```
@@ -0,0 +1,1208 @@
+			TotalTokens:      totalTokens,
+			CreatedAt:        startTime,
+		})
+	}
+
+	return &synthesis, nil
+}
+
+func (r *LibrarianRunner) saveMOC(ctx context.Context, cluster ClusterCandidate, synthesis *MOCSynthesisResponse) error {
+	topicTitle := strings.TrimSpace(synthesis.TopicTitle)
+	if topicTitle == "" {
+		topicTitle = strings.Title(strings.ReplaceAll(cluster.Tag, "-", " "))
```

golangci-lint (2.12.2)

[error] SA1019: strings.Title has been deprecated since Go 1.18 and an alternative has been available since Go 1.0: The rule Title uses for word boundaries does not handle Unicode punctuation properly. Use golang.org/x/text/cases instead. 

(staticcheck)